libraries/AP_HAL_ChibiOS/RCOutput.cpp

```diff
@@ -1563,7 +1565,7 @@ bool RCOutput::serial_write_bytes(const uint8_t *bytes, uint16_t len)
  */
 void RCOutput::serial_bit_irq(void)
 {
-    systime_t now = chVTGetSystemTimeX();
+    systime_t now = AP_HAL::micros();
```

Comment:
we always set system tick to be equal to micros, is that why this is ok?

Reply:
yes, but it should be uint32_t usec, I'll fix